sign_moduel.js

```diff
@@ -0,0 +1,39 @@
+export function validateEmail(emailInput, emailError) {
+  var email = emailInput.value;
+  if (email.length === 0) {
+    emailError.textContent = '이메일을 입력해주세요.';
+    emailError.style.display = 'block';
+    emailInput.classList.add('input-error');
+  } else if (!email.includes('@')) {
+    emailError.textContent = '올바른 이메일 주소가 아닙니다.';
+    emailError.style.display = 'block';
+    emailInput.classList.add('input-error');
+  } else {
+    emailError.style.display = 'none';
+    emailInput.classList.remove('input-error');
+  }
```

Comment:
```suggestion
  if (email.length === 0) {
    emailError.textContent = '이메일을 입력해주세요.';
    emailError.style.display = 'block';
    emailInput.classList.add('input-error');
    
    return;
  }
  
  if (!email.includes('@')) {
    emailError.textContent = '올바른 이메일 주소가 아닙니다.';
    emailError.style.display = 'block';
    emailInput.classList.add('input-error');
    
    return;
  }
  
  emailError.style.display = 'none';
  emailInput.classList.remove('input-error');
```

[증요도: 중간] 이런 early return 방식도 추천 드립니다. 불필요하게 else if문으로 여러개 가는 것 보다 특정 상황에서 더이상 진행할 필요 없을 경우에 주로 사용하고 특정 if문으로 끝나면 바로 이탈시키는 방식이라 이렇게 쓰는 것도 편합니다.

Reply:
감사합니다! 아직 효율적인 코드 작성에 대해서 생각하기엔 역량이 부족한 것 같습니다. 더 공부하겠습니다!